api-schema/xdb.yaml

```diff
@@ -114,6 +114,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ApiErrorResponse'
+  /api/v1/xdb/service/process-execution/rpc:
+    post:
+      summary: execute a RPC method of a process execution
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/ProcessExecutionRpcRequest'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ProcessExecutionRpcResponse'
+        '400':
+          description: invalid request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiErrorResponse'
+        '404':
```

Comment:
We also want to return different user/application side status code for:
* Global Attribute operation failure like unique key conflict or column doesn't exists etc. I have used 424 for this in start API. 
* RPC execution error from worker : Maybe 417 (expectation failure) for this one

Reply:
Expectation failutre seems to be related to the `Expect` field in the header.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/417

But I didn't find another better code, so will use this one for now.